mirror_network.go

```diff
@@ -5,16 +5,18 @@ import (
 )
 
 type _MirrorNetwork struct {
-	networkNodes map[string]*_MirrorNode
-	network      []string
-	index        uint
+	networkNodes      map[*ManagedNodeAddress]*_MirrorNode
```

Comment:
Should not use `*ManagedNodeAddress` as map key, use `string` instead. The key for `MirrorNetwork` should be the address, and the key for `Network` should be the serialized form of `AccountID`